src/main/java/gov/nasa/pds/harvest/search/HarvestSearchLauncher.java

```diff
@@ -595,10 +591,10 @@ private void backupOutputDirectory(File outputDir) throws IOException {
 	private void processMain(String[] args) 
 	{
 		// This removes the log4j warnings
-		ConsoleAppender ca = new ConsoleAppender(new PatternLayout("%-5p %m%n"));
-		ca.setThreshold(Priority.FATAL);
-
-		BasicConfigurator.configure(ca);
+//		ConsoleAppender ca = new ConsoleAppender(new PatternLayout("%-5p %m%n"));

```

Comment:
Rather than comment-out lines, just nuke 'em! (If you really want to keep them, then add a comment explaining why.)